etabotsite/etabotapp/TMSlib/ETAreport_placeholder.py

```diff
@@ -0,0 +1,74 @@
+"""Reports."""
+
+import pandas as pd
+import numpy as np
+
+
+def generate_status_report(ETApredict_obj, **kwargs):
+    """Generate total report.
+
+    Arguments:
+    ETApredict_obj - ETApredict class object, must have task_list.
+
+    Returns: 
+    json with hierarchical information.
+
+    """
+    report = {
```

Comment:
This report is hand crafted data, should only appear in unit/integration tests folder solely for testing purpose instead of in production code.

Reply:
in our use case we need to have this in placeholder module. in production placeholders are replaced with real functionality